File: URI/Py/1045.py
a,b,c = map(float, input().split())
d = [a,b,c]
d.sort(reverse=True)
a = d[0]
b = d[1]
c = d[2]

#print(str(a)+" "+str(b)+" "+str(c))

if a>= b+c:
    print("NAO FORMA TRIANGULO")
else:
    if a**2 == (b**2)+ (c**2):
        print("TRIANGULO RETANGULO")
    if a**2 > (b**2) + (c**2):
        print("TRIANGULO OBTUSANGULO")
    if a**2 < (b**2) + (c**2):
        print("TRIANGULO ACUTANGULO")
    if a==b==c:
        print("TRIANGULO EQUILATERO")
    if (a==b and b!=c) or (b==c and c!=a) or (a==c and c!=b):
        print("TRIANGULO ISOSCELES")

File: URI/C _ C++/Beginner/1045.c
#include <stdio.h>

int main(void)
{
    double a,b,c,x,y,z;

    scanf("%lf %lf %lf", &a,&b,&c);

    if(a>=b && a>=c) {
        if(b>=c) {
            x=a;
            y=b;
            z=c;
        }
        else {
            x=a;
            y=c;
            z=b;
        }
    }
    else if(b>=a && b>=c) {
        if(a>=c) {
            x=b;
            y=a;
            z=c;
        }
        else{
            x=b;
            y=c;
            z=a;
        }
    }
    else if(c>=a && c>=b) {
        if(a>=b) {
            x=c;
            y=a;
            z=b;
        }
        else {
            x=c;
            y=b;
            z=a;
        }
    }

    if(x>=y+z) printf("NAO FORMA TRIANGULO\n");
    else if((x*x)==((y*y)+(z*z))) printf("TRIANGULO RETANGULO\n");
    else if((x*x)>((y*y)+(z*z))) printf("TRIANGULO OBTUSANGULO\n");
    else if((x*x)<((y*y)+(z*z))) printf("TRIANGULO ACUTANGULO\n");
    if(x==y && y==z) printf("TRIANGULO EQUILATERO\n");
    else if(x==y || y==z) printf("TRIANGULO ISOSCELES\n");

    return 0;
}
